Popup/explainer.md

```diff
@@ -49,40 +49,69 @@ We propose a new HTML element called `popup`. This new element can be used for a
 
 `popup` will include:
 
-* A `show()` JavaScript method, for invoking the `popup`. When visible, the `popup` will have a default set of behaviors as well as a default positioning scheme.
+* A few options to invoke/show the `popup`:
+  * A `popup` attribute, applied to whichever element should invoke a given popup (if applicable).
+  * An optional `option` attribute, applied to the `popup` to express that it should be shown.
+  * A `show()` JavaScript method, for invoking the `popup`.
+* When visible, a default set of behaviors as well as a default positioning scheme.
 * Logic for an optional `autofocus` attribute which enables moving focus to the `popup` or to a descendent.
 * An optional `delegatesfocus` attribute, for passing focus to descendants.
 * An optional `anchor` attribute, which both relates the `popup` to an activating element and can be used in a separately-proposed, CSS-based anchor positioning scheme.
-* A `hide()` method for hiding the `popup`
-* [Light dismiss](#light-dismiss) behaviors.
+* A couple means to dismiss the `popup`:
+  * An optional `hidden` attribute, applied to the `popup` to express that it should be hidden.

```

Comment:
I don't disagree with you, but I will say that the addition of the `open` attribute is in large part responding to a request for parity with `dialog`. That element also includes a `hidden` attribute and advises authors to use it instead of removing `open`: [`dialog` in the HTML spec](https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element). If we port over the `open` attribute we should probably achieve full parity, even if it's a little weird (the alternative being, can we adjust `dialog` behavior? Would that cause compat issues?).

Reply:
It'd be ideal to adjust dialog behavior IMO. That's https://github.com/whatwg/html/issues/5802

Reply:
Well, there you go! Will update to reflect the ideal and maybe just make a note of [current] disparity somewhere. Thanks to both!

Reply:
Fixed, with a note in the appendix: https://github.com/MicrosoftEdge/MSEdgeExplainers/pull/459/commits/3e7994e940efe431abc66fc66c7120e5e8e521c6